### ISSUE TITLE
Adding Aria Value text to Hand Position Sliders 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#105]
Adding new Aria Value text to the hand sliders. Two new tables under Object & Context Responses heading in the [PDOM](https://docs.google.com/document/d/1HJvvGWYaUv9tlat9S7omQQVng8fGB4FuUhTEM1qx1z8/edit#heading=h.ue3a9evp2qmj) have all the new description plus examples.

**Comment by zepumph** [comment]
This is implemented on master. Only for the individual hands (the sliders), not the combined interaction. Furthermore please see https://github.com/phetsims/ratio-and-proportion/issues/104 for work to be done on incorrect alerts and the lack of alerts during keyboard input.

@terracoda and @Matthew-Moore240, you can test things out in https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-dev.43/phet/ratio-and-proportion_all_phet.html

This was a very simple implementation because it uses the exact same strings as the screen summary. Let me know if you need it to get more complicated.

**Comment by Matthew-Moore240** [comment]
Verified and looks good! Closing.
